src/codeu/chat/relay/ServerFrontEnd.java

```diff
@@ -35,72 +35,72 @@
   private static final Serializer<Relay.Bundle.Component> COMPONENT_SERIALIZER =
       new Serializer<Relay.Bundle.Component>() {
 
-    @Override
-    public Relay.Bundle.Component read(InputStream in) throws IOException {
-
-      final Uuid id = Uuid.SERIALIZER.read(in);
-      final String text = Serializers.STRING.read(in);
-      final Time time = Time.SERIALIZER.read(in);
-
-      // I could have passed the relay and use its "pack" method but that would
-      // have been more work than just building an object here.
-      return new Relay.Bundle.Component() {
-        @Override
-        public Uuid id() { return id; }
         @Override
-        public String text() { return text; }
+        public Relay.Bundle.Component read(InputStream in) throws IOException {
```

Comment:
I don't know why this is signaling whitespace differences when it is the same file used in [Google's master branch](https://github.com/google/codeu_project_2017/blob/master/src/codeu/chat/relay/ServerFrontEnd.java). I suppose that due to IntelliJ's reformatting there was a change in a branch that is now showing the different whitespaces.